tests/setup/test_interactive_env_setup.py

```
@@ -0,0 +1,511 @@
+
+    @patch("getpass.getpass", return_value="secret_password")
+    def test_prompt_input_secret_mode(self, mock_getpass):
+        """Test input with secret=True uses getpass"""
+        result = prompt_input("Enter password", secret=True)
+
+        assert result == "secret_password"
+        mock_getpass.assert_called_once()
+
+    @patch("builtins.input", return_value="")
+    @patch("getpass.getpass", return_value="")
+    def test_prompt_input_secret_required_empty(self, mock_getpass, mock_input):
```

Ruff (0.14.4)

Unused method argument: `mock_getpass`

(ARG002)